src/Symfony/Component/Cache/Traits/PhpFilesTrait.php

```diff
@@ -23,8 +24,13 @@
  */
 trait PhpFilesTrait
 {
-    use FilesystemCommonTrait;
+    use FilesystemCommonTrait {
+        doClear as private doCommonClear;
+        doDelete as private doCommonDelete;
+    }
 
+    private $expiries = array();
+    private $values = array();
     private $includeHandler;
```

Comment:
is it good idea force caching restored values? 
If I need large object for a short time - a can get it from cache and unset then.
If I need caching- I can chain with array cache.

Reply:
Behavior now opt-in via a new constructor argument. This is not the same as chaining with ArrayAdapter because ArrayAdapter has to serialize to deep clone, which nullifies the benefits of the Php* adapters.

Reply:
ArrayCache can store values without serializaing.

example:
long running service: `$cache->get('key');`
+ 5 sec
some script an same node: `$cache->set('key', $newData);`
+ 5 sec
long running service: `$cache->get('key'); // dont see changes`

It's not about ttl. It's about changes visibility.

Some caches has request locality (ArrayCache)
Some caches has node locality (File*, Opcache)
Some caches has cluster locality (Pdo, Memcached, Redis).

We mix caches when optimizing io, network, memory or cpu. Do you want to change PhpFilesCache locality?